client/rest-high-level/src/main/java/org/elasticsearch/client/ml/DeleteExpiredDataRequest.java

```diff
@@ -19,16 +19,77 @@
 package org.elasticsearch.client.ml;
 
 import org.elasticsearch.client.Validatable;
+import org.elasticsearch.common.unit.TimeValue;
+import org.elasticsearch.common.xcontent.ToXContentObject;
+import org.elasticsearch.common.xcontent.XContentBuilder;
+
+import java.io.IOException;
+import java.util.Objects;
 
 /**
  * Request to delete expired model snapshots and forecasts
  */
-public class DeleteExpiredDataRequest implements Validatable {
+public class DeleteExpiredDataRequest implements Validatable, ToXContentObject {
 
+    static final String REQUESTS_PER_SECOND = "requests_per_second";
+    static final String TIMEOUT = "timeout";
+    private final Float requestsPerSecond;
+    private final TimeValue timeout;
    /**
      * Create a new request to delete expired data
      */
     public DeleteExpiredDataRequest() {
+        this(null, null);
+    }
+
+    public DeleteExpiredDataRequest(Float requestsPerSecond, TimeValue timeout) {
+        this.requestsPerSecond = requestsPerSecond;
+        this.timeout = timeout;
+    }
+
+    /**
+     * The requests allowed per second in the underlying Delete by Query requests executed.
+     *
+     * `-1.0f` indicates the default behavior where throttling scales according to the number of data nodes.
```

Comment:
"default" is probably the wrong word for this comment.  The other HLRC docs say the default is `null`, and that's what's implemented in the HLRC.

So maybe "default" to "standard nightly maintenance behavior" or something like that.

Reply:
Definitely, saying `default` is a bit disingenuous.